starknet_py/net/models/deployer_addresses.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+from starknet_py.net.networks import Network, MAINNET, TESTNET
+
+
+def deployer_address_from_network(
```

Comment:
I'm not a fan of this function. From what I see it is used only once in the actual implementation and always `None` as the `deployer_address` will be passed. Maybe we could change it somehow